### PR TITLE
Vector tooltips infowindow

### PR DESCRIPTION
**Pull request by donflopez**
It depends on #1624 and fix https://github.com/CartoDB/tangram.cartodb/issues/8

### CODE REVIEW

File: src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js [hunk 1]
```diff
@@ -20,7 +21,44 @@ LeafletCartoDBWebglLayerGroupView.prototype = _.extend(
...
+        },
+        click: function (e) {
+          if (e.feature) {
+            self.trigger('featureClick', self._getFeatureObject(e));
+          }
+        }
+      });
     },
+    _getFeatureObject: function (e) {
+      var index = 0;
```
Comment (alonsogarciapablo) [inline src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js]: Why is this always `0`?

Comment (donflopez) [inline src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js]: I forgot to change it in the refactor! Good catch :)

File: src/geo/cartodb-layer-group.js [hunk 2]
```diff
@@ -39,6 +39,12 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     return this._getGroupedLayers()[index];
   },
 
+  getCartoLayerById: function (id) {
+    return this._layersCollection.filter(function (ly) {
```
Comment (alonsogarciapablo) [inline src/geo/cartodb-layer-group.js]: Perhaps we can use `Collection#get` -> http://backbonejs.org/#Collection-get

**Comment by alonsogarciapablo** [comment]
:+1:

**Pull request merged**